### PR TITLE
fix toggle box animation

### PR DESCRIPTION
**Pull request by dzonidoo**
link: authoring react - fix setting embargo #4619

### CODE REVIEW

File: app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx [hunk 1]
```diff
@@ -22,10 +26,31 @@ export class CustomHeaderToggleBox extends React.PureComponent<IPropsCustomHeade
...
+            this.contentRef.current.removeEventListener('animationend', this.handleAnimationEnd);
+        }
+    }
+
     render() {
-        let classes = classNames('sd-shadow--z1 new-collapse-box', {
+        const classes = classNames('sd-shadow--z1 new-collapse-box', {
             'new-collapse-box--open': this.state.isOpen,
         });
+        const childrenClasses = classNames('new-collapse-box__content-inner p-2 pt-0-5', {
```
Comment (tomaskikutis) [inline app-typescript/components/ToggleBox/CustomHeaderToggleBox.tsx]: `childrenClasses` doesn't sound like a great name. How about inlining the code so there's no confusion where it is applied?

File: app-typescript/components/ToggleBox/SimpleToggleBox.tsx [hunk 2]
```diff
@@ -106,7 +102,7 @@ export class SimpleToggleBox extends React.PureComponent<IPropsSimple, IState> {
                     {badge ? badge : null}
                 </a>
                 <div className="toggle-box__content-wraper">
-                    <div id={this.htmlId} className={childrenClasses} aria-hidden={!isOpen} ref={this.contentRef}>
+                    <div id={this.htmlId} className={'toggle-box__content' + (this.state.isAnimating && ' toggle-box__content--animation')} aria-hidden={!isOpen} ref={this.contentRef}>
```
Comment (tomaskikutis) [inline app-typescript/components/ToggleBox/SimpleToggleBox.tsx]: Didn't I tell you not to use boolean operators when you're not trying to get a boolean result? (except JSX shorthand)

run `'classname1' + (false && ' classname2')`

Comment (dzonidoo) [reply app-typescript/components/ToggleBox/SimpleToggleBox.tsx]: ...so:
'toggle-box__content' + (this.state.isAnimating ? ' toggle-box__content--animation' : ' ')?

Comment (tomaskikutis) [reply app-typescript/components/ToggleBox/SimpleToggleBox.tsx]: using that classnames function is fine too, I didn't ask not to use it. I was asking not to create a variable

Comment (dzonidoo) [reply app-typescript/components/ToggleBox/SimpleToggleBox.tsx]: ahh i miss that point

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**